src/neo-vm/Utility.cs

```diff
@@ -0,0 +1,16 @@
+using System.Text;
+
+namespace Neo.VM
+{
+    internal static class Utility
+    {
+        public static Encoding StrictUTF8 { get; }
```

Comment:
If we change to public, we could reuse this in core

Reply:
I also thought about this, but I couldn't find a suitable place to put it. `Utility` is not suitable for `public`.